python/src/deltachat/account.py

```diff
@@ -31,38 +161,17 @@ def shutdown(self):
         # interrupt idle calls do not seem to release the
         # blocking call to smtp|imap idle. This means we
         # also can't now close the database because the
-        # threads might still need it
+        # threads might still need it.
         # capi.lib.dc_close(self.dc_context)
 
-    def process_event(self, ctx, evt_name, data1, data2):
+    def _process_event(self, ctx, evt_name, data1, data2):
```

Comment:
When reading this function I don't actually know what's a cffi/c object and what's a python object.  I think it's important to make the distinction very clearly and have a clear boundary so that I don't have to wonder about this.  E.g. is `_evtlogger` taking cffi objects or python ones?  I feel like it should be python ones since converting should happen as soon as it's possible to do so accurately.

Reply:
yes, these are python objects here.  the py_dc_callback converts them early on. so all occurences of evt_name, data1, data2 etc. in account.py are python objects.